packages/core/saved-objects/core-saved-objects-migration-server-internal/src/zdt/model/stages/init.ts

```diff
@@ -127,7 +127,10 @@ export const init: ModelStage<
     aliases: existingAliases,
     aliasActions,
     previousMappings: currentMappings,
-    previousAlgorithm: currentAlgo === 'v2-compatible' ? ('v2' as const) : ('zdt' as const),
+    previousAlgorithm:
+      currentAlgo === 'v2-compatible' || currentAlgo === 'v2-partially-migrated'
+        ? ('v2' as const)
+        : ('zdt' as const),
```

Comment:
I consider `v2-partially-migrated` as `v2` here, as the only usage of this info is for the documentation migration, where we always trigger a document scan if `previousAlgorithm === 'v2'`.